tools/cellxgene_census_builder/src/cellxgene_census_builder/__main__.py

```diff
@@ -58,6 +58,9 @@ def do_build(args: CensusBuildArgs, skip_completed_steps: bool = False) -> int:
         do_build_soma,
         do_validate_soma,
         do_create_reports,
```

Comment:
We should also sync reports, and I don't think that's currently handled.

Reply:
this was not in the build process the last time I checked, but is easily added.

Reply:
It wasn't, I just realized the omission today while doing the release follow-up tasks. Ty!

Reply:
added